src/microbe_stage/MicrobeWorldSimulation.cs

```diff
@@ -216,6 +217,7 @@ public void Init(Node visualDisplayRoot, CompoundCloudSystem cloudSystem, IMicro
         osmoregulationAndHealingSystem = new OsmoregulationAndHealingSystem(EntitySystem, couldParallelize);
         pilusDamageSystem = new PilusDamageSystem(EntitySystem, couldParallelize);
         slimeSlowdownSystem = new SlimeSlowdownSystem(cloudSystem, EntitySystem, couldParallelize);
+        mucocystSystem = new MucocystSystem(EntitySystem, couldParallelize);
```

Comment:
Did you test the performance of parallelizing the mucocyst system? I ask because the mucocyst system will run on *each* cell on each game update so it could be more performant to use multiple threads to run through all the entities. Though I guess as the AI won't know how to turn on the mucocyst the system will be very branch preditor friendly so it might run as fast without threads.

Reply:
This didn't get a reply yet.